src/main/kotlin/frc/robot/Robot.kt

```diff
@@ -128,7 +128,9 @@ object Robot : LoggedRobot() {
 
     /** This function is called once when teleop is enabled.  */
     override fun teleopInit() {
-        autonomousCommand.cancel()
+        if (::autonomousCommand.isInitialized) {
```

Comment:
I'm not near a computer, but shouldn't it be 
```suggestion
        if (autonomousCommand.isInitialized()) {
```
Also, I don't see `Command` has `isInitialized`, I think you need to use `isScheduled()`.

Reply:
No, Barak is correct, if you use `autonomousCommand.isInitialized()` the uninitialized command will get called and throw an error. Using `::` only references it.
`isInitialized()` is a property of lateinit variables.

Reply:
Ohhh ok. I was thinking about command initialization. Thanks for the explanation!

Reply:
Copilot is a great teacher🤩